tests/units/online/test_multiple_connections.py

```diff
@@ -54,3 +54,10 @@ def test_same_robot(reachy_sdk_zeroed: ReachySDK) -> None:
         assert np.allclose(reachy_sdk_zeroed.l_arm.get_current_positions(), reachy2.l_arm.get_current_positions(), atol=1)
     assert np.allclose(reachy_sdk_zeroed.l_arm.get_current_positions(), move1_goal, atol=1e-03)
     assert np.allclose(reachy2.l_arm.get_current_positions(), move1_goal, atol=1e-03)
+
+
+@pytest.mark.timeout(2)
```

Comment:
no need for marker timeout (I guess you mean this https://pypi.org/project/pytest-timeout/) if we have the timeout in the while.